src/main/java/com/prototype/booking/Application.java

```diff
@@ -0,0 +1,23 @@
+package com.prototype.booking;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.integration.config.EnableIntegration;
+
+@SpringBootApplication
+@ComponentScan
+@EnableIntegration
+public class Application extends SpringBootServletInitializer {
```

Comment:
Please move the **Application** contents to **BookAMeetingRoomApplication** file.

Reply:
Yes, I will move.